src/WCSimDetectorConfigs.cc

```diff
@@ -1387,7 +1387,7 @@ void WCSimDetectorConstruction::SetNuPrismBeamTest_16cShort_mPMTGeometry()
 {
   WCDetectorName = "NuPRISMBeamTest_16cShort_mPMT";
   WCIDCollectionName = WCDetectorName +"-glassFaceWCPMT";
-	mPMT_ID_PMT = "PMT3inchR12199_02";    //can be changed in macro through mPMT settings.
+	mPMT_ID_PMT = "PMT3inchWCTE";    //can be changed in macro through mPMT settings.
```

Comment:
I don't actually see where this setting is described in WCTE.mac. I'd recommend adding instructions there (and specifying what the default is)